governance/meeting-host-playbook.md

```diff
@@ -0,0 +1,84 @@
+# TAG Environmental Sustainability Meeting Host Playbook
+
+Meetings are held to coordinate and discuss TAG ENV activities. The TAG ENV leads host these meetings. The following playbook can be used for reference to structure the meeting, you don't need to stick step by step (word by word), it should give guidance how a meeting usually is structured.
+
+## Requirements to host the meeting as Release Team Lead / Lead Shadow
+
+* Install Zoom and check your settings (especially audio and video).
+* Get access to the [CNCF TAG YouTube Channel](https://www.youtube.com/@CNCFEnvTAG) to start the livestream.
+
+## Pre-Meeting
+* Add a new blank agenda entry to the meeting notes by copying the template and editing the details accordingly.
+* Announce the meeting about an hour before in the `#tag-env-sustainability` Slack channel.
+
+```
+Hi folks! The CNCF TAG ENV bi-weekly meeting takes place 1 hour from now
+Links:
+* Agenda & Minutes: https://bit.ly/cncf-tag-env-meeting-notes
+* Zoom Link: https://zoom.us/my/cncftagenvsustainability, passcode `77777`
+
+If anyone has anything they’d like to discuss, please add to it to the agenda, thanks!
+```
+
+## During the Meeting
+
+### Preparation
+
+* Slack reminder (thread) *"The meeting start now!"*
+* [Join](https://zoom.us/my/cncftagenvsustainability) the meeting
+* Turn on your camera
+* Say hello to everyone 
+  * *"Hello all, let's wait a few minutes for everyone to join"*
+* Post the meeting notes to the meeting chat
+  * *"I posted the meeting notes link to chat, please add yourself to the attendees list, thank you!"*
+* Prepare the YouTube livestream
+* Open meeting notes and share screen
+* Wait until 3 minutes past
+  * *"Since it is now 3 minutes past, we will get started"*
+  * *"If you rather not get recorded, you can now turn off your video"*
+* Start the YouTube livestream
+
+### Intro
+
+* Welcome everyone
+  * *"Hello everyone, my name is 'Alice' and I will be the host for this CNCF TAG Environmental Sustainability meeting"*
+* Call out the date for the record
+  *  *"Today is 'Monday' 'May' the '4th'"*
+* Mention the CoC 
+  * *"Please be aware that this is a CNCF community meeting which falls under the CNCF code of conduct, which can be summarized to be excellent to each other, thank you all!"*
+* Recording warning 
+  * *"This meeting is recorded and will be publicly posted to YouTube so please be mindful what you say is being recorded."*
+
+### Agenda
+
+* Introductions
+  * "I am pleased that you are all here. Before we continue with the recurring topics of the meeting, I would like to open the floor to anyone who would like to introduce themselves to the group and perhaps describe why they are interested in the TAG."
+* WG updates
```

Comment:
Before updates, we might add a section for special guests e.g. introducing other foundations, projects or, if we have a presentation to go with them first

Reply:
> special guests e.g. introducing other foundations

is part of the first point Introductions

> presentation

is part of the open discussion items

Reply:
we can switch WG updates with Open discussion items if we feel its better for the flow

Reply:
> we can switch WG updates with Open discussion items if we feel its better for the flow

I think the host can decide this on the meeting. We can be flexible about it. In general the structure Introduction -> WG Updates -> Open Discussion items sound good to me

Reply:
The only reason to swap introduction and WG updates is so new people have a moment to get their bearings and/or get zoom to behave properly, but I am fine whatever order.

Reply:
I think we can leave it in the current form